guides/generic.md

```diff
@@ -0,0 +1,79 @@
+- [Generic Client](#generic-client)
+  - [Getting the Client](#get-client)
+  - [Sending Simple Requests](#request-bodyless)
+  - [Sending JSON Requests](#request-json)
+  - [Sending JSON using POJOs](#request-pojo)
+
+# Generic Client
+
+There are rare circumstances when the typed OpenSearch client APIs are too constraining and there is a need to communicate with OpenSearch cluster (or individual nodes) over generic HTTP request / response communication. The `OpenSearchGenericClient` could be used in such cases.
+
+## Getting the Client
+The following sample code gets the `OpenSearchGenericClient` rom the `OpenSearchClient` instance.
+
+```java
+final OpenSearchGenericClient generic = javaClient().generic();
+```
+
+## Sending Simple Request
+The following sample code sends a simple request that does not require any payload to be provided.
+
+```java
+// compare with what the low level client outputs
+try (Response response = javaClient().generic().execute(Requests.builder().endpoint("/").method("GET").build())) {
+   // ...
+}
+```
+
+Please notice that the `Response` instance should be closed explicitly in order to free up any allocated resource (like response input streams or buffers). The generic client never interprets status codes and provides the direct access to the response as it was received from the server. This is responsibility of the caller to decide what should happen in case of unsuccessful invocations.
```

Comment:
There are two statements here that are sort of independent. I would break it up with two separate examples on their own paragraphs.